httputils/writers.go

```diff
@@ -0,0 +1,12 @@
+package httputils
```

Comment:
stdlib naming of `httptest` and `io/util` would direct us along the lines of using `httputil` - and what am I missing with `ErrorF`, wouldn't it be `Errorf`?

Reply:
updated